definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -196,9 +196,11 @@ declare module 'express' {
     use(middleware: Middleware): this;
     use(...middelware: Array<Middleware>): this;
     use(path: string, ...middelware: Array<Middleware>): this;
+    use(path: string, router: Router): this;
 
     route(path: string): Route;
   }
+  declare type $Router = Router;
```

Comment:
Yea, so just to clarify: You can use `import type` with any thing that is a "type". Classes are a special kind of thing that is simultaneously a type *and* a value -- so it's ok to use `import type` on classes.